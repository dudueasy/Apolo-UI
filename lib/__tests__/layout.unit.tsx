import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../layout/layout';
import {shallow} from 'enzyme';


describe('Layout component', ()=>{
  test('should render properly', function () {
    const component = shallow(<Layout className={'hello'}/>)

    const layoutWrapper = component.find('[className="apoloUI-layout hello"]')

    expect(layoutWrapper.exists()).toBeTruthy()
    console.log(layoutWrapper.debug())
  });

  test('should match snapshot', function () {
      const tree = renderer.create(<Layout className={'hello'}/>).toJSON()
      expect(tree).toMatchSnapshot()
  });
})



