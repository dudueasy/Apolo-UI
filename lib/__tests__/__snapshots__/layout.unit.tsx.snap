// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should match snapshot 1`] = `
<div
  className="apoloUI-layout hello"
/>
`;
